src/stylesheets/elements/form-controls/_date-picker.scss

```diff
@@ -45,7 +45,7 @@
 
 .usa-date-picker__button {
   @extend %usa-date-picker__button;
-  @include add-background-svg("calendar-alt-solid");
+  @include add-background-svg("usa-icons/calendar_today");
```

Comment:
Height seems to be smaller than previous.

Reply:
I can't reproduce this one

Reply:
Seems to be a Firefox only issue. The difference is only half a pixel.